### ISSUE TITLE
Error message when a temporary file failed to be removed on Ubuntu, related to #150

### ISSUE THREAD

**Issue by joseph2rs** [geopython/pywps#218]
# Description

I'have an error that should be a consequence of some modifications made for #150 :
```
Traceback (most recent call last):
  File "/home/afsy/jrs/Documents/geometwps/srvgeomet27/src/pywps-master/pywps/app/Process.py", line 227, in _run_process
    100, STATUS.DONE_STATUS, clean=self.async)
  File "/home/afsy/jrs/Documents/geometwps/srvgeomet27/src/pywps-master/pywps/app/WPSResponse.py", line 70, in update_status
    self.write_response_doc(self.doc, clean)
  File "/home/afsy/jrs/Documents/geometwps/srvgeomet27/src/pywps-master/pywps/app/WPSResponse.py", line 84, in write_response_doc
    self.process.clean()
  File "/home/afsy/jrs/Documents/geometwps/srvgeomet27/src/pywps-master/pywps/app/Process.py", line 281, in clean
    except WindowsError as err:
NameError: global name 'WindowsError' is not defined
```

# Environment

- operating system: Ubuntu 14.04, 16.04
- Python version: 2.7
- PyWPS version: 4, latest
- source/distribution
 - [x] git clone
 - [ ] Debian
 - [ ] PyPI
 - [ ] zip/tar.gz
 - [ ] other (please specify): 
- web server
 - [ ] Apache/mod_wsgi
 - [ ] CGI
 - [x] other (please specify): Flask

# Steps to Reproduce
Lock on the tmp dir
# Additional Information

**Comment by jachym** [comment]
Hi,

the error is due to missing WindwsError class on non-Windows system

I've prepared fix for that https://github.com/jachym/PyWPS/commit/00e5b11f2368c551890be8f0d82c28c925cb464b#diff-bddcbbd3f8d8f4182a33f03a1f9a847dL278

Quick solution: just remove the two lines in Process.py 

```
-        except WindowsError as err:
-                LOGGER.error('Windows Error: %s', err)
```

**Comment by ldesousa** [comment]
Fixed in PR #222
